content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
 complexes but may be less effective since on average protein complexes have
 fewer sequence homologs.
 
+### Protein-Protein Interactions
+
+Protein-protein interactions (PPIs) are highly specific and non-accidental physical contacts between proteins which occur for purposes other than generic protein production or degradation [@doi:10.1371/journal.pcbi.1000807].
+PPIs are key to many cellular processes like metabolism and immune responses.
```

Comment:
PPIs are involved in almost all cellular processes.  Perhaps we could cut this line?  I'm looking for places to shorten the text.